canonical-web-and-design/maas-ui
# Handle closing hardware menu

## Done
- Updated the shared workspace to use the same linting as the react project.
- Fix prettier/lint issues.
- Handling close the hardware menu when clicking outside or clicking on an item.

## QA
- Resize your window so that the hardware option appears in the header.
- Open the menu then click outside it, it should close.
- Open the menu and then click on one of the menu items, the menu should close.

## Fixes
Fixes: https://github.com/canonical-web-and-design/maas-ui/issues/1052.
